passwordless.go

```diff
@@ -113,11 +113,28 @@ func (p *Passwordless) RequestToken(ctx context.Context, s, uid, recipient strin
 	}
 }
 
+// PostVerifyAction is an action to take after validation has succesfully occured.
+type PostVerifyAction func(ctx context.Context, s TokenStore, uid, token string, valid bool) (bool, error)
```

Comment:
Unclear on the purpose of the bool in the return values here. When would it ever be different to the incoming `valid` value?